src/index.js

```diff
@@ -2,27 +2,68 @@
 
 var MutationObserver = window.MutationObserver
 
+// @TODO, remove post testing
+var testGroups = [
+  {
+    name: 'old-timerz',
+    key: 'github.com/dino-dna/tagadactyl',
+    global: false,
+    users: [
+      { login: 'dylancwood' },
+      { login: 'rkelly' }
+    ]
+  },
+  {
+    name: 'def',
+    key: 'github.com/dino-dna/NOTTDACTYL',
+    global: false,
+    users: [
+      { login: 'swashcap' },
+      { login: 'wa11-e' }
+    ]
+
+  }
+]
 var tdac = {
-  _cache: {},
+  _cache: {
+    collaborators: {}, // GH collaborators, as stated by GH service. e.g. { 'dino-dna/tagadactyl': [...] }
+    groups: [] // user specified groups, e.g. [{ key: 'dino-dna/tagadactyl', name: 'mygroup', users: [...], global: boolean }]
+  },
+  get currentProjectKey () { return window.location.hostname + '/' + this.owner + '/' + this.repo },
+  get owner () { return window.location.pathname.split('/')[1] },
+  get repo () { return window.location.pathname.split('/')[2] },
+
+  bindClickConfigureGroupHandler (group) {
+    var node = document.getElementById(this.mapGroupToGroupId(group))
+    node.addEventListener('click', function (evt) {
+      evt.stopImmediatePropagation()
+      evt.stopPropagation()
+      evt.preventDefault()
+      console.log('blah!')
```

Comment:
hey @tophf, thx for the tips!  in fact, i eventually traced it down to a mousedown handler!  tricky tricky